contracts/farm/MasterChef_Polygon.sol

```diff
@@ -303,12 +308,18 @@ contract MasterChef_Polygon is Upgradeable {
             multiplier.mul(GOVPerBlock).mul(pool.allocPoint).div(
                 totalAllocPoint
             );
-        coordinator.mint(devaddr, GOVReward.div(1e19));
-        coordinator.mint(address(this), GOVReward.div(1e18));
-        pool.accGOVPerShare = pool.accGOVPerShare.add(
-            GOVReward.div(1e6).div(lpSupply)
-        );
-        pool.lastRewardBlock = block.number;
+        // 250m = 250 * 1e6
+        if (coordinator.totalMinted() < 250*1e6*1e18) {
```

Comment:
there is a drawback here: last mint can actually buypass 250m cap by last amount. so in my test for example totalMinted was `250000000000000008375164659`

Reply:
if we want to limit ALL minting to 250m maybe move all that to `minCoordinator` ?